lago/virt.py

```diff
@@ -86,7 +86,6 @@ def __init__(self, prefix, vm_specs, net_specs):
 
         libvirt_url = config.get('libvirt_url')
         self.libvirt_con = libvirt_utils.get_libvirt_connection(
```

Comment:
I think that the `virt` object shouldn't be aware of provider-specific details, thus the `libvirt_con` reference (and all the code that use it) can be removed from `virt.py`.